core/src/main/scala/kanaloa/reactive/dispatcher/queue/QueueProcessor.scala

```diff
@@ -116,11 +118,10 @@ trait QueueProcessor extends Actor with ActorLogging with MessageScheduler {
     case _ ⇒ //Ignore
   }
 
-  private def createWorker(index: Int): WorkerRef = {
-    val timestamp = LocalDateTime.now.toInstant(ZoneOffset.UTC).toEpochMilli
+  private def createWorker(): WorkerRef = {
     val worker = context.actorOf(
       workerProp(queue),
-      s"worker-${queue.path.name}-$index-${timestamp}"
+      s"worker-${workerCount.incrementAndGet()}"
```

Comment:
Meaning, since this Workers are created during the Receive function(and not on a Thread/Future), the calls are synchronous.  Do we need an AtomicInt, or can we just use a regular monotonically increasing int?